ofl/ddcuchen/METADATA.pb

```diff
@@ -0,0 +1,17 @@
+name: "DDC Uchen"
```

Comment:
Per the naming requirements in https://github.com/google/fonts/blob/master/CONTRIBUTING.md we may want to remove the DDC

Reply:
done